<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>668</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>668</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Powered by n+e</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/fig/sudoku</normaloff>:/fig/sudoku</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>404</width>
      <height>404</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Game">
    <property name="title">
     <string>&amp;New Game</string>
    </property>
    <addaction name="actionRandom_5"/>
    <addaction name="actionCustom"/>
    <addaction name="actionNew_Game"/>
   </widget>
   <addaction name="menuNew_Game"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInfo"/>
   <addaction name="actionPause"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionReset"/>
   <addaction name="actionMusic"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/sudoku</normaloff>:/fig/sudoku</iconset>
   </property>
   <property name="text">
    <string>info</string>
   </property>
   <property name="toolTip">
    <string>Infomation</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/prev</normaloff>:/fig/prev</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/previous</normaloff>:/fig/previous</iconset>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
   <property name="toolTip">
    <string>Reset to initial</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/pause</normaloff>:/fig/pause</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/next</normaloff>:/fig/next</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionMusic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/fig/mute</normaloff>:/fig/mute</iconset>
   </property>
   <property name="text">
    <string>music</string>
   </property>
   <property name="toolTip">
    <string>Play music</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionRandom_5">
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
